docs/api-contracts/Disputes.md

```diff
@@ -112,4 +104,14 @@ type DisputeCreateResponse = string;
     - `Authorization: Bearer <JWT>`
 
 **NOTE:** This endpoint involves files being exchanged. This requires the use of `multipart/form-data` instead of JSON.
-This is left incomplete until this exception is documenteda.
\ No newline at end of file
+This is left incomplete until this exception is documenteda.
+
+# Dispute Status Change
+- **Endpoint:** `PUT /dispute/status_update`
```

Comment:
To maintain consistency, shouldn't it rather be `update-status`? Or even just `status`, considering the *update* part is encoded by using the `put` method